src/env.ts

```diff
@@ -88,3 +89,15 @@ export function loadBranchName() {
 
     if (process.env.TRACE) console.log("set branch", parsed);
 }
+
+export function loadLogValues() {
+    const defaultLogLevel = "info";
+    const defaultLogFile = "logs/log.json";
```

Comment:
hi @khoanguyen102 , isnt that we already set the default values in .env files for LogLevel and LogFile, not sure why we need this here, is there any specific reason?

Reply:
thanks @datlam162 for your question. In case of other projects reference this library, the .env file to be loaded as from that project instead of this .env from library. If they forgot putting these values, the default will be used